manuscript.Rmd

```diff
@@ -424,30 +457,73 @@ In that case, the readers can be certain, that they have received a complete acc
+Incentive structures must be more willing to evaluate not the outcomes of a study but the rigor with which it was conducted [@caganSanFranciscoDeclaration2013; @schonbrodtResponsibleResearchAssessment2022].
+Journal editors could encourage theoretical developments and be willing to accept registered reports [@friedLackTheoryBuilding2020; @friedTheoriesModelsWhat2020; @vanrooijTheoryDevelopmentRequires2020; @vanrooijTheoryTestHow2021].
+Funding agencies should demand an explicit statement about theoretical risk in relation to detectability and must be willing to provide the necessary resources to reach adequate levels of both [@kooleRewardingReplicationsSure2012].
+
+Our later result, on the importance of preregistration for minimizing uncertainty, has two important implications.
+The first is, that even if all imaginable actions regarding promoting higher theoretical risk are taken, confirmatory research should be preregistered.
+Otherwise the uncertainty about the theoretical risk will diminish the advantage of confirmatory research.
+Second, even under less than ideal circumstances for confirmatory research, preregistration is beneficial.
+Preregistering exploratory studies increases the expected epistemic value by virtue of reducing uncertainty about theorethical risk.
+Nevertheless, exploratory studies will have a lower expected epistemic value than a more confirmatory study, if both are preregistered and have equal detectability.
+
+Focusing on uncertainty reduction also explains two common practices of preregistration that do not align with a confirmatory research agenda.
+First, researchers seldomly predict precise numerical outcomes, instead they use preregistrations to describe the process that generates the results.
+Precise predictions would have very high theoretical risk (they are likely incorrect if the theory is wrong).
+A statistical procedure, may have high or low theoretical risk depending on the specifics of the model used.
+Specifying the process, therefore, is in line with the rational we propose here, but is incompatible with strictly confirmatory research.
+
+Second, researchers often have to deviate from the preregistration and make data-dependent decisions outside of the preregistration.
+If the only goal of preregistration is to ensure confirmatory research, such changes are not justifiable.
+However, under our rational some changes may be justified.
+Any change would increase the uncertainty about the theoretical risk and may even decrease the theorethical risk.
+The changes still may be worthwhile, if the negative outcomes may be offset by an increase in detectability due to the change.
+Consider a preregistration that failed to specify what to do with missing values where the researchers subsequently encounter missings.
+In such case, the detectability is zero, since the data cannot be analyzed without a decision about how to handle the missing data.
```

Comment:
```suggestion
In such case, detectability becomes zero because the data cannot be analyzed without a post-hoc decision about how to handle the missing data.
```